### PR TITLE
Cleanup old folders on central

### PR DESCRIPTION
**Pull request by charleskawczynski**
In light of #2988, this PR cleans up old jobs on central, when we move new data there. Let's wait until #2988 is resolved before merging this, however.

### CODE REVIEW

File: regression_tests/move_output.jl [hunk 1]
```diff
@@ -46,3 +46,34 @@ if get(ENV, "BUILDKITE_PIPELINE_SLUG", nothing) == "climaatmos-ci"
...
 end
+
+get_ref_counter(p) = parse(Int, first(readlines(joinpath(p, "ref_counter.jl"))))
+function cleanup_central(cluster_data_prefix)
+    @warn "Cleaning up old files on central"
+    # Get (sorted) array of paths, `pop!(sorted_paths)`
+    # is the most recent merged folder.
+    sorted_paths = sorted_dataset_folder(; dir = cluster_data_prefix)
+    if !isempty(sorted_paths)
+        N = length(sorted_paths) - 10
```
Comment (Sbozzolo) [inline regression_tests/move_output.jl:57]: Maybe adding something like this to make clear what 10 and 5 represent?
```suggestion
keep_latest_n = 10
keep_latest_ref_counters = 5
```

**Review by Sbozzolo — APPROVED** [review]

**Comment by Sbozzolo** [comment]
Looking at the output of the CI jobs, the reference folder contains a lot of files that probably should not be there, such as,
```
[ Info:    File:`dmcpupcpuicpu__2emcpupcpuicpu__2fmcpupcpuicpu__2amcpupcpuicpu__2umcpupcpuicpu__2lmcpupcpuicpu__2tmcpupcpuicpu__2_mcpupcpuicpu__2cmcpupcpuicpu__2omcpupcpuicpu__2nmcpupcpuicpu__2fmcpupcpuicpu__2i_g`
[ Info:    File:`dmgpupgpuigpu__4emgpupgpuigpu__4fmgpupgpuigpu__4amgpupgpuigpu__4umgpupgpuigpu__4lmgpupgpuigpu__4tmgpupgpuigpu__4_mgpupgpuigpu__4cmgpupgpuigpu__4omgpupgpuigpu__4nmgpupgpuigpu__4fmgpupgpuigpu__4i_g`
```

<details>
  <summary>Full list</summary>

```
[ Info: Files on main:
[ Info:    File:`018f4de1-40b2-4153-a5f1-efd157043bcd`
[ Info:    File:`018f4de1-40b4-4a2b-987d-533f5cc28dbc`
[ Info:    File:`018f4de1-40b7-4c91-9417-26f21a616594`
[ Info:    File:`018f4de1-40b9-4892-947b-91eabbbab4b8`
[ Info:    File:`018f4de1-40bc-478f-8926-4d05a67a8b4c`
[ Info:    File:`018f4de1-40cb-48a4-a9ed-9e185ffcae58`
[ Info:    File:`018f4de1-40ce-4c9e-94c2-800c60dd80c5`
[ Info:    File:`018f4de1-40d1-4bbe-859f-5131d18f199a`
[ Info:    File:`018f4de1-40d3-42ad-a7dd-1e49ccacc445`
[ Info:    File:`018f4de1-40d6-4258-8bd3-0fbfb30fce13`
[ Info:    File:`018f4de1-40d9-4d78-991c-ae49a51e4a3a`
[ Info:    File:`018f4de1-40db-4eb7-a32c-3076464f0569`
[ Info:    File:`018f4de1-40de-438f-aa4f-67032608ce09`
[ Info:    File:`018f4de1-40e0-4a63-8bcf-59fa19a96b3a`
[ Info:    File:`018f4de1-40e3-4a80-ad96-5a2da1fbd7a2`
[ Info:    File:`018f4de1-40e6-4706-bcb7-1470c31a07eb`
[ Info:    File:`018f4de1-40ed-4860-ace8-910ebe0849df`
[ Info:    File:`018f4de1-40f0-4d77-bcaf-2fab89eb2aca`
[ Info:    File:`018f4de1-40f2-49dd-a78d-a69132e7a1a3`
[ Info:    File:`018f4de1-40f4-4297-91bd-02773c338b08`
[ Info:    File:`018f4de1-40f7-4a37-ab27-411158bf1654`
[ Info:    File:`018f4de1-40fc-4f95-a925-c3dae6a0f717`
[ Info:    File:`018f4de1-40ff-43ee-a180-331eb410a3c4`
[ Info:    File:`018f4de1-4104-42e8-b76c-a304839c1e08`
[ Info:    File:`018f4de1-4106-40d3-9b52-1148e018c243`
[ Info:    File:`018f4de1-410d-4b92-99ab-72da1bfaaa0a`
[ Info:    File:`018f4de1-410f-44d6-b526-3cdfe2183da9`
[ Info:    File:`018f4de1-4111-4e31-9278-fcc81f9c68f2`
[ Info:    File:`018f4de1-4114-4e83-8c68-80f73cf5aa41`
[ Info:    File:`018f4de1-4117-498f-9e25-e585641ba2e0`
[ Info:    File:`018f4de1-411c-41d3-a562-b780b4cb490b`
[ Info:    File:`018f4de1-411e-412e-ad9c-5f765e03242b`
[ Info:    File:`018f4de1-4124-4a56-8e3d-f5a6493ae546`
[ Info:    File:`018f4de1-4126-462c-b459-3eb79eb71264`
[ Info:    File:`018f4de1-412b-4904-8c26-d2804b7ee417`
[ Info:    File:`018f4de1-412e-4254-8ef1-ac4291fefdb7`
[ Info:    File:`018f4de1-4130-4264-99d3-1e677969f07a`
[ Info:    File:`018f4de1-4133-435f-a61a-dd943716513d`
[ Info:    File:`018f4de1-4136-4516-986c-cf03b8e910c0`
[ Info:    File:`018f4de1-4138-4b7a-93ca-2992f93b9508`
[ Info:    File:`018f4de1-413b-40ac-b28a-818516b02e6b`
[ Info:    File:`018f4de1-413d-4403-8ffd-12f15060335a`
[ Info:    File:`018f4de1-413f-45b0-8fa7-4d986a498f7a`
[ Info:    File:`018f4de1-4142-4a1f-8fe5-c47606b02bb7`
[ Info:    File:`018f4de1-4147-43bc-9f5e-9affcb9238c0`
[ Info:    File:`018f4de1-414a-4e6c-bc81-340f08c1856e`
[ Info:    File:`018f4de1-414c-43e2-a23d-33431275e3df`
[ Info:    File:`018f4de1-414e-4a44-a7bb-aa7551c73111`
[ Info:    File:`018f4de1-4151-498c-ba0e-02ed322b3f99`
[ Info:    File:`018f4de1-4153-4aad-bfb9-06836a371a1e`
[ Info:    File:`018f4de1-4156-4897-a83d-76ee6b2e24b1`
[ Info:    File:`018f4de1-4159-4c80-8796-b949afcfb68b`
[ Info:    File:`018f4de1-415b-4968-80c1-b55d03b6190a`
[ Info:    File:`018f4de1-415d-431a-b3de-7bbf9b69c82c`
[ Info:    File:`018f4de1-4160-4bd9-8c87-7547990840b0`
[ Info:    File:`018f4de1-4162-46ea-9fcd-7bd6c0292d17`
[ Info:    File:`018f4de1-4165-4f7c-93cd-630ddb405f39`
[ Info:    File:`018f4de1-4167-4172-a1fa-621ea7d88c87`
[ Info:    File:`018f4de1-416a-43d8-9a26-91e679fe146e`
[ Info:    File:`018f4de1-4171-4626-87a4-b55217f253ab`
[ Info:    File:`018f4de1-4173-47c6-af0c-c34ff836f62e`
[ Info:    File:`018f4de1-4176-46b7-91ed-2314c6f15f75`
[ Info:    File:`018f4de1-4178-4c87-b9f1-98c40c29f320`
[ Info:    File:`018f4de1-417a-4dc9-a816-0c44bacf5645`
[ Info:    File:`018f4de1-417d-4c6f-90b6-97ecbd72e745`
[ Info:    File:`018f4de1-417f-499e-9e9e-346093f7e75c`
[ Info:    File:`018f4de1-41ae-4c98-9858-ef0574456d67`
[ Info:    File:`aquaplanet_rhoe_equil_clearsky_tvinsol_0M_slabocean`
[ Info:    File:`aquaplanet_rhoe_equil_clearsky_tvinsol_0M_slabocean_1_proc_cpu`
[ Info:    File:`box_density_current_test`
[ Info:    File:`box_density_current_test_1_proc_cpu`
[ Info:    File:`box_hydrostatic_balance_rhoe`
[ Info:    File:`box_hydrostatic_balance_rhoe_1_proc_cpu`
[ Info:    File:`central_cloud_diag_gpu_hs_rhoe_equil_55km_nz63_0M`
[ Info:    File:`central_cloud_diag_gpu_hs_rhoe_equil_55km_nz63_0M_1_proc_gpu`
[ Info:    File:`central_gpu_hs_rhoe_equil_55km_nz63_0M`
[ Info:    File:`central_gpu_hs_rhoe_equil_55km_nz63_0M_1_proc_gpu`
[ Info:    File:`deep_sphere_baroclinic_wave_rhoe_equilmoist`
[ Info:    File:`default_config`
[ Info:    File:`default_config_1_proc_cpu`
[ Info:    File:`default_config_1_proc_gpu`
[ Info:    File:`default_config_1_proc_threaded_cpu_8`
[ Info:    File:`default_config_mpi_2_cpu`
[ Info:    File:`default_config_mpi_4_gpu`
[ Info:    File:`diagnostic_edmfx_aquaplanet`
[ Info:    File:`diagnostic_edmfx_bomex_box`
[ Info:    File:`diagnostic_edmfx_bomex_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_bomex_stretched_box`
[ Info:    File:`diagnostic_edmfx_bomex_stretched_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_dycoms_rf01_box`
[ Info:    File:`diagnostic_edmfx_dycoms_rf01_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_dycoms_rf01_explicit_box`
[ Info:    File:`diagnostic_edmfx_dycoms_rf01_explicit_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_gabls_box`
[ Info:    File:`diagnostic_edmfx_gabls_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_rico_box`
[ Info:    File:`diagnostic_edmfx_rico_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_test_box`
[ Info:    File:`diagnostic_edmfx_test_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_trmm_box`
[ Info:    File:`diagnostic_edmfx_trmm_box_0M`
[ Info:    File:`diagnostic_edmfx_trmm_box_0M_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_trmm_box_1_proc_cpu`
[ Info:    File:`diagnostic_edmfx_trmm_stretched_box`
[ Info:    File:`diagnostic_edmfx_trmm_stretched_box_1_proc_cpu`
[ Info:    File:`dmcpupcpuicpu__2emcpupcpuicpu__2fmcpupcpuicpu__2amcpupcpuicpu__2umcpupcpuicpu__2lmcpupcpuicpu__2tmcpupcpuicpu__2_mcpupcpuicpu__2cmcpupcpuicpu__2omcpupcpuicpu__2nmcpupcpuicpu__2fmcpupcpuicpu__2i_g`
[ Info:    File:`dmgpupgpuigpu__4emgpupgpuigpu__4fmgpupgpuigpu__4amgpupgpuigpu__4umgpupgpuigpu__4lmgpupgpuigpu__4tmgpupgpuigpu__4_mgpupgpuigpu__4cmgpupgpuigpu__4omgpupgpuigpu__4nmgpupgpuigpu__4fmgpupgpuigpu__4i_g`
[ Info:    File:`gpu_aquaplanet_dyamond`
[ Info:    File:`gpu_aquaplanet_dyamond_1_proc_gpu`
[ Info:    File:`gpu_baroclinic_wave_rhoe`
[ Info:    File:`gpu_baroclinic_wave_rhoe_1_proc_gpu`
[ Info:    File:`gpu_diagnostic_edmfx_aquaplanet`
[ Info:    File:`gpu_diagnostic_edmfx_aquaplanet_1_proc_gpu`
[ Info:    File:`gpu_prognostic_edmfx_aquaplanet`
[ Info:    File:`gpu_prognostic_edmfx_aquaplanet_1_proc_gpu`
[ Info:    File:`invalidations`
[ Info:    File:`longrun_bw_rhoe_highres_100`
[ Info:    File:`longrun_bw_rhoe_highres_1080`
[ Info:    File:`longrun_bw_rhoe_highres_120`
[ Info:    File:`longrun_bw_rhoe_highres_1200`
[ Info:    File:`longrun_bw_rhoe_highres_150`
[ Info:    File:`longrun_bw_rhoe_highres_180`
[ Info:    File:`longrun_bw_rhoe_highres_200`
[ Info:    File:`longrun_bw_rhoe_highres_300`
[ Info:    File:`longrun_bw_rhoe_highres_360`
[ Info:    File:`longrun_bw_rhoe_highres_400`
[ Info:    File:`longrun_bw_rhoe_highres_450`
[ Info:    File:`longrun_bw_rhoe_highres_480`
[ Info:    File:`longrun_bw_rhoe_highres_50`
[ Info:    File:`longrun_bw_rhoe_highres_540`
[ Info:    File:`longrun_bw_rhoe_highres_60`
[ Info:    File:`longrun_bw_rhoe_highres_600`
[ Info:    File:`longrun_bw_rhoe_highres_640`
[ Info:    File:`longrun_bw_rhoe_highres_720`
[ Info:    File:`longrun_bw_rhoe_highres_80`
[ Info:    File:`longrun_bw_rhoe_highres_800`
[ Info:    File:`longrun_bw_rhoe_highres_90`
[ Info:    File:`longrun_bw_rhoe_highres_900`
[ Info:    File:`longrun_bw_rhoe_highres_960`
[ Info:    File:`mpi_make_restart`
[ Info:    File:`mpi_make_restart_mpi_2_cpu`
[ Info:    File:`mpi_sphere_aquaplanet_rhoe_equilmoist_clearsky`
[ Info:    File:`mpi_sphere_aquaplanet_rhoe_equilmoist_clearsky_mpi_2_cpu`
[ Info:    File:`perf_benchmarks.json`
[ Info:    File:`plane_agnesi_mountain_test_stretched`
[ Info:    File:`plane_agnesi_mountain_test_stretched_1_proc_cpu`
[ Info:    File:`plane_agnesi_mountain_test_uniform`
[ Info:    File:`plane_agnesi_mountain_test_uniform_1_proc_cpu`
[ Info:    File:`plane_density_current_test`
[ Info:    File:`plane_density_current_test_1_proc_cpu`
[ Info:    File:`plane_schar_mountain_kinetic_energy_check_stretched`
[ Info:    File:`plane_schar_mountain_test_stretched`
[ Info:    File:`plane_schar_mountain_test_stretched_1_proc_cpu`
[ Info:    File:`plane_schar_mountain_test_uniform`
[ Info:    File:`plane_schar_mountain_test_uniform_1_proc_cpu`
[ Info:    File:`prep_remap`
[ Info:    File:`prep_remap_mpi_2_cpu`
[ Info:    File:`prognostic_edmfx_adv_test_column`
[ Info:    File:`prognostic_edmfx_adv_test_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_aquaplanet`
[ Info:    File:`prognostic_edmfx_aquaplanet_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_bomex_box`
[ Info:    File:`prognostic_edmfx_bomex_box_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_bomex_column`
[ Info:    File:`prognostic_edmfx_bomex_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_bomex_explicit_column`
[ Info:    File:`prognostic_edmfx_bomex_explicit_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_bomex_fixtke_column`
[ Info:    File:`prognostic_edmfx_bomex_fixtke_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_bomex_stretched_column`
[ Info:    File:`prognostic_edmfx_bomex_stretched_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_dycoms_rf01_column`
[ Info:    File:`prognostic_edmfx_dycoms_rf01_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_gabls_column`
[ Info:    File:`prognostic_edmfx_gabls_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_gcmdriven_column`
[ Info:    File:`prognostic_edmfx_gcmdriven_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_rico_column`
[ Info:    File:`prognostic_edmfx_rico_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_simpleplume_column`
[ Info:    File:`prognostic_edmfx_simpleplume_column_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_trmm_column`
[ Info:    File:`prognostic_edmfx_trmm_column_0M`
[ Info:    File:`prognostic_edmfx_trmm_column_0M_1_proc_cpu`
[ Info:    File:`prognostic_edmfx_trmm_column_1_proc_cpu`
[ Info:    File:`ref_counter.jl`
[ Info:    File:`restart_mpi_baroclinic_wave_rhoe`
[ Info:    File:`single_column_hydrostatic_balance_ft64`
[ Info:    File:`single_column_hydrostatic_balance_ft64_1_proc_cpu`
[ Info:    File:`single_column_nonorographic_gravity_wave`
[ Info:    File:`single_column_nonorographic_gravity_wave_1_proc_cpu`
[ Info:    File:`single_column_precipitation_test`
[ Info:    File:`single_column_precipitation_test_1_proc_cpu`
[ Info:    File:`single_column_radiative_equilibrium_allsky_idealized_clouds`
[ Info:    File:`single_column_radiative_equilibrium_allsky_idealized_clouds_1_proc_cpu`
[ Info:    File:`single_column_radiative_equilibrium_clearsky`
[ Info:    File:`single_column_radiative_equilibrium_clearsky_1_proc_cpu`
[ Info:    File:`single_column_radiative_equilibrium_clearsky_prognostic_surface_temp`
[ Info:    File:`single_column_radiative_equilibrium_clearsky_prognostic_surface_temp_1_proc_cpu`
[ Info:    File:`single_column_radiative_equilibrium_gray`
[ Info:    File:`single_column_radiative_equilibrium_gray_1_proc_cpu`
[ Info:    File:`sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric`
[ Info:    File:`sphere_aquaplanet_rhoe_equilmoist_allsky_gw_res`
[ Info:    File:`sphere_aquaplanet_rhoe_equilmoist_allsky_gw_res_1_proc_cpu`
[ Info:    File:`sphere_aquaplanet_rhoe_nonequilmoist_allsky_gw_res`
[ Info:    File:`sphere_aquaplanet_rhoe_nonequilmoist_allsky_gw_res_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe`
[ Info:    File:`sphere_baroclinic_wave_rhoe_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_cldiag`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_cldiag_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_conservation_ft64`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_conservation_ft64_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_expvdiff`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_expvdiff_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_impvdiff`
[ Info:    File:`sphere_baroclinic_wave_rhoe_equilmoist_impvdiff_1_proc_cpu`
[ Info:    File:`sphere_baroclinic_wave_rhoe_hughes2023`
[ Info:    File:`sphere_baroclinic_wave_rhoe_topography_dcmip_rs`
[ Info:    File:`sphere_baroclinic_wave_rhoe_topography_dcmip_rs_1_proc_cpu`
[ Info:    File:`sphere_held_suarez_rhoe_equilmoist_hightop_sponge`
[ Info:    File:`sphere_held_suarez_rhoe_equilmoist_topography_dcmip`
[ Info:    File:`sphere_held_suarez_rhoe_equilmoist_topography_dcmip_1_proc_cpu`
[ Info:    File:`sphere_held_suarez_rhoe_hightop`
[ Info:    File:`sphere_held_suarez_rhoe_hightop_1_proc_cpu`
[ Info:    File:`sphere_held_suarez_rhoe_topography_dcmip`
[ Info:    File:`sphere_held_suarez_rhoe_topography_dcmip_1_proc_cpu`
[ Info:    File:`sphere_hydrostatic_balance_rhoe_ft64`
[ Info:    File:`sphere_hydrostatic_balance_rhoe_ft64_1_proc_cpu`
[ Info:    File:`sphere_ssp_baroclinic_wave_rhoe_equilmoist_dcmip200`
[ Info:    File:`sphere_ssp_baroclinic_wave_rhoe_equilmoist_dcmip200_1_proc_cpu`
[ Info:    File:`sphere_ssp_baroclinic_wave_rhoe_equilmoist_earth`
[ Info:    File:`sphere_ssp_baroclinic_wave_rhoe_equilmoist_earth_1_proc_cpu`
[ Info:    File:`target_gpu_implicit_baroclinic_wave`
[ Info:    File:`target_gpu_implicit_baroclinic_wave_1_proc_gpu`
[ Info:    File:`target_gpu_implicit_baroclinic_wave_4process`
[ Info:    File:`target_gpu_implicit_baroclinic_wave_4process_mpi_4_gpu`
[ Info:    File:`test_invalidations`
[ Info:    File:`test_io`
[ Info:    File:`test_io_1_proc_threaded_cpu_8`
[ Info:    File:`test_mpi_io`
[ Info:    File:`test_mpi_io_mpi_2_cpu`
```
</details>

It looks like that every PR (even ones that are for testing or that don't get merged) add files to the reference folder, which I think is not indended behavior (or, is it?).

**Comment by charleskawczynski** [comment]
> Looking at the output of the CI jobs, the reference folder contains a lot of files that probably should not be there, such as,
> 
> ```
> [ Info:    File:`dmcpupcpuicpu__2emcpupcpuicpu__2fmcpupcpuicpu__2amcpupcpuicpu__2umcpupcpuicpu__2lmcpupcpuicpu__2tmcpupcpuicpu__2_mcpupcpuicpu__2cmcpupcpuicpu__2omcpupcpuicpu__2nmcpupcpuicpu__2fmcpupcpuicpu__2i_g`
> [ Info:    File:`dmgpupgpuigpu__4emgpupgpuigpu__4fmgpupgpuigpu__4amgpupgpuigpu__4umgpupgpuigpu__4lmgpupgpuigpu__4tmgpupgpuigpu__4_mgpupgpuigpu__4cmgpupgpuigpu__4omgpupgpuigpu__4nmgpupgpuigpu__4fmgpupgpuigpu__4i_g`
> ```
> 
> Full list
> It looks like that every PR (even ones that are for testing or that don't get merged) add files to the reference folder, which I think is not indended behavior (or, is it?).

Yeah, it's not the intended behavior. I opened #2988 recently to fix this.

**Comment by Sbozzolo** [comment]
> > Looking at the output of the CI jobs, the reference folder contains a lot of files that probably should not be there, such as,
> > ```
> > [ Info:    File:`dmcpupcpuicpu__2emcpupcpuicpu__2fmcpupcpuicpu__2amcpupcpuicpu__2umcpupcpuicpu__2lmcpupcpuicpu__2tmcpupcpuicpu__2_mcpupcpuicpu__2cmcpupcpuicpu__2omcpupcpuicpu__2nmcpupcpuicpu__2fmcpupcpuicpu__2i_g`
> > [ Info:    File:`dmgpupgpuigpu__4emgpupgpuigpu__4fmgpupgpuigpu__4amgpupgpuigpu__4umgpupgpuigpu__4lmgpupgpuigpu__4tmgpupgpuigpu__4_mgpupgpuigpu__4cmgpupgpuigpu__4omgpupgpuigpu__4nmgpupgpuigpu__4fmgpupgpuigpu__4i_g`
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Full list
> > It looks like that every PR (even ones that are for testing or that don't get merged) add files to the reference folder, which I think is not indended behavior (or, is it?).
> 
> Yeah, it's not the intended behavior. I opened #2988 recently to fix this.

Thanks! I didn't see the issue

**Pull request merged**